docker/salt/tethyscore.sls

```diff
@@ -149,7 +156,8 @@ Create_Database_User_and_SuperUser_TethysCore:
         -N "{{ TETHYS_DB_SUPERUSER }}"
         -P "{{ TETHYS_DB_SUPERUSER_PASS }}"
     - shell: /bin/bash
-    - unless: /bin/bash -c "[ -f "{{ TETHYS_PERSIST }}/setup_complete" ];"
+    - unless: /bin/bash -c "[ -f '{{ TETHYS_PERSIST }}/setup_complete' ];"
```

Comment:
There is the case of wanting to use the Tethys Docker to connect to an existing PostgreSQL database, in which case the `TETHYS_DB_ENGINE` would be `'django.db.backends.postgresql'` but you would not want to create these databases. I've taken the approach of adding a separate variable `SKIP_DB_SETUP` to be able to control this independently of the `TETHYS_DB_ENGINE` value. 

```salt
    - unless: /bin/bash -c "[ -f "{{ TETHYS_PERSIST }}/setup_complete" ] || [ "$SKIP_DB_SETUP" = true ];"
```

Reply:
added conditional for all the different db settings when setting up all the portal config settings. In addition, added, the SKIP_DB_SETUP env variable: 
fcd5a9140214a0d0978c6a1a2a7aee8eee1773f6